lcls-twincat-pmps/PMPS/MajorComponents/PMPS_MPS_Interface/FB_BeamClassWatcher.TcPOU

```diff
@@ -21,8 +21,8 @@ VAR_INPUT
     i_stCurrentBeamParams    :    ST_BeamParams; //Link to global beam params
     i_stMachineTargetBeamParams    :    ST_BeamParams; //Link to global machine target beam params
     i_stRequestedBeamParams    :    ST_BeamParams; //Link to arbiter output or beam param. requestor
-    // Reset fault
-    i_xReset: BOOL;
+    //Auto Reset fault
+    i_xAutoReset: BOOL;
```

Comment:
This is technically an API break, but the original value didn't have any effect, so it's probably OK (and probably good to remove unused inputs anyway)

Reply:
This function block is being used by the arbiter. There have been discussion by TMO about auto resetting this fault. While we haven't made that decision yet, I thought to push it with the new library in case we needed it.